upgrade/steps/SystemChecksStep.class

```diff
@@ -34,6 +34,7 @@ class SystemChecksStep extends UpgradeStep {
     function _getBytes($val) {
 	$val = trim($val);
 	$last = $val[strlen($val)-1];
+    $val = intval(substr($val, 0, strlen($val) - 1));
```

Comment:
Hmm, I think this is going to have a problem if the value doesn't end in "g", "m", or "k". For example, if $val starts off as "23", this will set it to "2".

One way to fix this would be to add a `default` case to the switch statement that does

```$val = intval($val);```

Thanks!

Reply:
I will have another look at it! 

Thank you!